lcls-twincat-pmps/PMPS/HelperFunctions/F_DifferentBeamParams.TcPOU

```diff
@@ -26,6 +28,8 @@ END_VAR
       <ST><![CDATA[xAttOK := BeamParam1.nTran <> BeamParam2.nTran;
 xEvOK := BeamParam1.neVRange <> BeamParam2.neVRange;
 xRateOK     := BeamParam1.nRate <> BeamParam2.nRate;
+xBCOK	:= BeamParam1.nBCRange <> BeamParam2.nBCRange;// OR BeamParam1.nBeamClass <> BeamParam2.nBeamClass ; //Is this right?
```

Comment:
I think this is correct as-is and doesn't need the OR (responding to the comment)